### PR TITLE
Fix port config in single mode

### PR DESCRIPTION
**Pull request by bufrr**
### Proposed changes in this pull request
Explain the changes in this pull request in order to help the project maintainers understand the overall impact of it.

### Type (put an `x` where ever applicable)
- [x] Bug fix: Link to the issue
- [ ] Feature (Non-breaking change)
- [ ] Feature (Breaking change)
- [ ] Documentation Improvement

### CODE REVIEW

File: util/config/config.go [hunk 1]
```diff
@@ -539,11 +575,13 @@ func (config *Configuration) incrementPort() {
...
 		config.NodePort,
 		config.HttpWsPort,
+		config.HttpWssPort,
+		config.HttpsJsonPort,
 		config.HttpJsonPort,
 	}
 	minPort, maxPort := findMinMaxPort(allPorts)
 	step := maxPort - minPort + 1
-	var delta uint16
+	var delta uint16 = 0
```
Comment (yilunzhang) [inline util/config/config.go]: This is implicit for Go

File: util/config/config.go [hunk 2]
```diff
@@ -539,11 +575,13 @@ func (config *Configuration) incrementPort() {
 	allPorts := []uint16{
 		config.NodePort,
 		config.HttpWsPort,
+		config.HttpWssPort,
+		config.HttpsJsonPort,
```
Comment (gdmmx) [inline util/config/config.go]: Suggest adjust slice order, for `base+offset` coincident with port's actual purpose.
```
base_port + 1 = NodePort
base_port + 2 = WsPort
base_port + 3 = JsonPort
base_port + 4 = WssPort
base_port + 5 = sJsonPort
```

File: util/config/config.go (hunk 1 above)
Comment (bufrr) [reply util/config/config.go]: done

File: util/config/config.go (hunk 2 above)
Comment (bufrr) [reply util/config/config.go]: done

**Pull request merged**